tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,88 @@
 fig.gca().view_init(azim=70, elev=15)
 ten_twenty_montage.plot(kind='topomap', show_names=False)
 
+###############################################################################
+# .. _control-chan-projection:
+#
+# Controlling channel projection (MNE vs EEGLAB)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Channel positions in 2d space are obtained by projecting their actual 3d
+# positions onto a sphere. Because ``'standard_1020'`` montage contains
+# realistic, not spherical, channel positions, we will use a different montage
+# to demonstrate controlling how channels are projected to 2d space.
+
+biosemi_montage = mne.channels.make_standard_montage('biosemi64')
+biosemi_montage.plot(show_names=False)
+
+###############################################################################
+# By default a sphere  with an origin in ``(0, 0, 0)`` x, y, z coordinates and
+# radius of ``0.095`` meters (9.5 cm) is used. You can use a different sphere
+# radius by passing a single value to ``sphere`` argument in any function that
+# plots channels in 2d (like :meth:`~mne.channels.DigMontage.plot`that we use
+# here, but also for example :func:`mne.viz.plot_topomap`):
+
+biosemi_montage.plot(show_names=False, sphere=0.07)
+
+###############################################################################
+# To control not only radius, but also the sphere origin, pass a
+# ``(x, y, z, radius)`` tuple to ``sphere`` argument:
+
+biosemi_montage.plot(show_names=False, sphere=(0.03, 0.02, 0.01, 0.075))
+
+###############################################################################
+# In mne-python the head center and therefore the sphere center are calculated
+# using fiducial points. Because of this the head circle represents head
```

Comment:
Yes, that's better, thanks. But it seems I may still not understand something here - I'm not sure what you are referring to as "the automated sphere center". The default sphere center is `(0, 0, 0)`, right?

Reply:
[The docs](https://mne.tools/stable/generated/mne.channels.DigMontage.html#mne.channels.DigMontage.plot) say:

> The sphere parameters to use for the cartoon head. Can be array-like of shape (4,) to give the X/Y/Z origin and radius in meters, or a single float to give the radius (origin assumed 0, 0, 0). Can also be a spherical ConductorModel, which will use the origin and radius. Can be “auto” to use a digitization-based fit. Can also be None (default) to use ‘auto’ when enough extra digitization points are available, and 0.095 otherwise. Currently the head radius does not affect plotting.

So for `mon.plot` in theory there should be enough points (EEG points can be used) and so the origin should not be (0, 0, 0). If it is, we either have an implementation (probably this) or a documentation bug.

Reply:
at least I think it will try to use the EEG dig points, even though it says "extra digitization points"...

Reply:
Okay I was wrong, it does just look for `extra`:

https://github.com/mne-tools/mne-python/blob/af1120c4cdea7086c2af661fefd37ef93ac8f410/mne/utils/check.py#L650

IIRC the motivation was that when you apply a montage, missing channels are not included from `info['dig']`. I think if we kept them all in there (maybe with the others as extra points?) then checked for EEG+extra, we could make it always fit an origin.

We can look into enabling this -- I actually don't think it would be too much work in DigMontage -- but if what you have here works well enough already, then we should just stick with it (especially right before release) rather than try to make DigMontage do more.

Reply:
Defining head center (and sphere origin) based on fiducials is simple and makes sense. However, I'm not so sure about using all available dig and channel points for that: because then, depending on the distribution of these points, the head outline has a different meaning. I think it is good to have a consistent meaning for what the head outline represents. But I may still be missing something here.